lib/JIRA/Client/Automated.pm

```diff
@@ -252,9 +252,15 @@ sub _handle_error_response {
     my ($self, $response, $request) = @_;
 
     my $msg = $response->status_line;
-    $msg .= sprintf " %s", $response->decoded_content
+    #$msg .= sprintf " %s", $response->decoded_content
+    #    if $response->decoded_content;
+    use Data::Dump qw(pp);
```

Comment:
Can you remove debugging code from this and delete any commented out code that's been replaced?